ethereum/contracts/bridge/L1ERC20Bridge.sol

```diff
@@ -13,6 +13,8 @@ import "./interfaces/IL2ERC20Bridge.sol";
 import "./libraries/BridgeInitializationHelper.sol";
 
 import "../zksync/interfaces/IZkSync.sol";
+import {TxStatus} from "../zksync/interfaces/IMailbox.sol";
+import "../zksync/Storage.sol";
 import "../common/libraries/UnsafeBytes.sol";
```

Comment:
Should we use explicit imports everywhere?

Reply:
> Remove some of the implicit imports. This PR additionally creates some new implicit imports in another files, but those will be fixed in subsequent PRs

I described it in the title of the PR, I'll remove those in the different PRs dedicated to another files